crates/polars-arrow/src/bit_util.rs

```diff
@@ -79,3 +81,93 @@ pub fn ceil(value: usize, divisor: usize) -> usize {
         quot
     }
 }
+
+#[inline]
+fn get_zeroes(chunk: u64) -> u32 {
```

Comment:
Not sure if this is the best file to place it here.

Reply:
We can make this a `const fn`.

I think this is a better fit in `polars-utils` as it is very generic.

Reply:
> We can make this a const fn

Yes, `const fn` should work.

> I think this is a better fit in polars-utils as it is very generic.

`get_zeroes` and `get_ones` can be moved to `polars-utils`, but other methods like `first_set_bit` can not as they all depended on `arrow` which `polars-utils` not aware of.

Reply:
Uhh, isn't this implementation just wrong? Even if we were to support big-endian platforms, this is not correct. Endianness only swaps *byte* order, `trailing_zeros` vs `leading_zeros` swaps *bit* order.

Reply:
Good point @orlp. We were saved because of the dominance of little endiannes here. 

@reswqa can you simply remove those functions and use the ones define on the type.